packages/bot-web-ui/globals.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module '@deriv/account';
+declare module '@deriv/bot-skeleton';
+declare module '@deriv/cashier';
+declare module '@deriv/components';
+declare module '@deriv/appstore';
+declare module '@deriv/p2p';
+declare module '@deriv/trader';
+declare module '@deriv/translations';
+declare module '@deriv/shared';
+declare module '@deriv/bot-skeleton/src/constants/messages';
```

Comment:
do we need this line? `@deriv/bot-skeleton` is already covered in line 2.

Reply:
We don't need some modules here 
As I checked we only use these modules on the `bot-web-ui`
@deriv/deriv-charts
'@deriv/shared'
'@deriv/components'
'@deriv/bot-skeleton'
'@deriv/translations'